spec/spec_helper.rb

```diff
@@ -31,7 +31,10 @@
   end
 end
 
-Puppet::Util::Log.level = :info
-Puppet::Util::Log.newdestination(:console)
+def puppet_debug_override
+  return unless ENV['SPEC_PUPPET_DEBUG']
+  Puppet::Util::Log.level = :debug
+  Puppet::Util::Log.newdestination(:console)
+end
```

Comment:
is this really necessary? wouldn't it be the same just with:
```ruby
if ENV['SPEC_PUPPET_DEBUG']
    Puppet::Util::Log.level = :debug
    Puppet::Util::Log.newdestination(:console)
end
```
and do we really need to invoke this before each test?